### PR TITLE
Improvement Kubernetes docs 

### PR DESCRIPTION
**Pull request by Damenus**

### CODE REVIEW

File: examples/kubernetes/monitoring/wca/README.md [hunk 1]
```diff
@@ -1,74 +1,78 @@
...
+      ```
+      
+      Where names `node100 node101 node102` should be replaced by your kubernetes node names.
+      If you want deploy wca on all nodes, you can delete affinity in daemonset spec.
+
+
+2. Build image (from main project repo) and push to your registry
+
+   Build Docker image and push to private repo. You can use make command to this. Like in example below.
+   You have to replace `DOCKER_REPOSITORY_URL` variable to yours own docker registry.
```
Comment (ppalucki) [inline examples/kubernetes/monitoring/wca/README.md:35]: s/yours/your/

**Review by ppalucki — CHANGES_REQUESTED** [review]
some typos

File: examples/kubernetes/monitoring/wca/DEVEL.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+ADVANCE WCA DEPLOY
```
Comment (wacuuu) [inline examples/kubernetes/monitoring/wca/DEVEL.md:1]: Perhaps it should be ADVANCED?

File: examples/kubernetes/monitoring/wca/DEVEL.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+2. as daemonset, but using kubelet to scrape workloads (historical option)
+3. outside pod
+
+
+WCA USING KUBELET
+-----------
+
+Add private key and certificate to Secrets
+
+Workload Collocation Agent required private key and certificate to connect with kubelet.
```
Comment (wacuuu) [inline examples/kubernetes/monitoring/wca/DEVEL.md:16]: s/required/requires/

File: examples/kubernetes/monitoring/wca/DEVEL.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+Example how add this files to Secrets:
+
+.. code-block:: bash
+
+    sudo kubectl create secret generic kubelet-key-crt --from-file=./client.crt --from-file=./client.key --namespace=wca
+    
+    
+WCA OUTSIDE KUBERNETES
```
Comment (wacuuu) [inline examples/kubernetes/monitoring/wca/DEVEL.md:24]: WCA OUTSIDE OF KUBERNETES

File: examples/kubernetes/monitoring/wca/DEVEL.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+---------------
+
+Optionally, you can use token to connect to Kube API Server, while WCA is using outside pod.
```
Comment (wacuuu) [inline examples/kubernetes/monitoring/wca/DEVEL.md:27]: I don't get this part. WCA is running in a pod, yet it's not a part of kubernetes?

File: examples/kubernetes/monitoring/wca/README.md [hunk 6]
```diff
@@ -1,74 +1,78 @@
 Getting started
 ===============
 
-To run wca in daemon set: 
+Below instruction is about run wca as DaemonSet on cluster. This way uses kustomize to deploy all components.
+Kustomize is available from kubectl 1.14.
+
+1. Prepare cluster
+
+   Workload Collocation Agent required existing `wca` namespace and label `monitoring=wca` on nodes,
```
Comment (wacuuu) [inline examples/kubernetes/monitoring/wca/README.md:9]: requires wca namespace to exist

File: examples/kubernetes/monitoring/wca/README.md [hunk 7]
```diff
@@ -1,74 +1,78 @@
...
+   
+      Namespace can be crated by using kubectl by a following command:
+      
+      ```bash
+          kubectl create namespace wca
+      ```
+
+   2. Choose monitoring node 
+      
+      Label can be crated by using kubectl by a following command:
```
Comment (wacuuu) [inline examples/kubernetes/monitoring/wca/README.md:22]: I think it should be "by the following command"

File: examples/kubernetes/monitoring/wca/README.md [hunk 8]
```diff
@@ -1,74 +1,78 @@
...
+      
+      ```bash
+          kubectl label nodes node100 node101 node102 monitoring=wca
+      ```
+      
+      Where names `node100 node101 node102` should be replaced by your kubernetes node names.
+      If you want deploy wca on all nodes, you can delete affinity in daemonset spec.
```
Comment (wacuuu) [inline examples/kubernetes/monitoring/wca/README.md:29]: If you want **to** deploy

File: examples/kubernetes/monitoring/wca/README.md [hunk 9]
```diff
@@ -1,74 +1,78 @@
...
+
+5. Deploy wca
+   Finally use the command below to deploy all wca components.
+   
+   ```bash
+       kubectl apply -k examples/kubernetes/monitoring/wca
+   ```
+
+
+After deploy wca, you can deploy Prometheus operator to collect metrics from wca and visual them in Prometheus.
```
Comment (wacuuu) [inline examples/kubernetes/monitoring/wca/README.md:72]: After deploying wca

File: examples/kubernetes/monitoring/wca/README.md [hunk 10]
```diff
@@ -1,74 +1,78 @@
 Getting started
 ===============
 
-To run wca in daemon set: 
+Below instruction is about run wca as DaemonSet on cluster. This way uses kustomize to deploy all components.
```
[the remark this answers is not in the document]
Comment (Damenus) [reply examples/kubernetes/monitoring/wca/README.md:4]: fixed

File: examples/kubernetes/monitoring/wca/daemonset.yaml [hunk 11]
```diff
@@ -27,6 +27,7 @@ spec:
                   values:
                   - wca
                   - wca_cadvisor
+      # or debugging purpose, alternative way to nodeAffinity selection nodes
```
Comment (wacuuu) [inline examples/kubernetes/monitoring/wca/daemonset.yaml:30]: missing f in potential for in the beginning of the line?

File: examples/kubernetes/monitoring/wca/README.md [hunk 12]
```diff
@@ -1,74 +1,78 @@
...
+Kustomize is available from kubectl 1.14.
```
[the remark this answers is not in the document]
Comment (Damenus) [reply examples/kubernetes/monitoring/wca/README.md:5]: added

File: examples/kubernetes/monitoring/wca/README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (Damenus) [reply examples/kubernetes/monitoring/wca/README.md:9]: fixed

File: examples/kubernetes/monitoring/wca/README.md (hunk 1 above)
Comment (Damenus) [reply examples/kubernetes/monitoring/wca/README.md:35]: fixed

File: examples/kubernetes/monitoring/wca/DEVEL.md (hunk 2 above)
Comment (Damenus) [reply examples/kubernetes/monitoring/wca/DEVEL.md:1]: Fixed, it sounds better

File: examples/kubernetes/monitoring/wca/DEVEL.md (hunk 3 above)
Comment (Damenus) [reply examples/kubernetes/monitoring/wca/DEVEL.md:16]: fixed

File: examples/kubernetes/monitoring/wca/DEVEL.md (hunk 4 above)
Comment (Damenus) [reply examples/kubernetes/monitoring/wca/DEVEL.md:24]: fixed

File: examples/kubernetes/monitoring/wca/README.md (hunk 7 above)
Comment (Damenus) [reply examples/kubernetes/monitoring/wca/README.md:22]: fixed

File: examples/kubernetes/monitoring/wca/README.md (hunk 8 above)
Comment (Damenus) [reply examples/kubernetes/monitoring/wca/README.md:29]: fixed

File: examples/kubernetes/monitoring/wca/README.md (hunk 9 above)
Comment (Damenus) [reply examples/kubernetes/monitoring/wca/README.md:72]: fixed

File: examples/kubernetes/monitoring/wca/daemonset.yaml (hunk 11 above)
Comment (Damenus) [reply examples/kubernetes/monitoring/wca/daemonset.yaml:30]: yeap, fixed

File: examples/kubernetes/monitoring/wca/README.md [hunk 13]
```diff
@@ -1,74 +1,78 @@
...
+          kubectl label nodes node100 node101 node102 monitoring=wca
+      ```
+      
+      Where names `node100 node101 node102` should be replaced by your kubernetes node names.
+      If you want deploy wca on all nodes, you can delete affinity in daemonset spec.
+
+
+2. Build image (from main project repo) and push to your registry
+
+   Build Docker image and push to private repo. You can use make command to this. Like in example below.
```
[the remark this answers is not in the document]
Comment (Damenus) [reply examples/kubernetes/monitoring/wca/README.md:34]: removed

File: examples/kubernetes/monitoring/wca/DEVEL.md (hunk 5 above)
Comment (Damenus) [reply examples/kubernetes/monitoring/wca/DEVEL.md:27]: I mean that WCA isn't in K8S network (don't get certs and token, cannot connect to services which work only inside the network) and you must prepare all as like as you outside the cluster

**Review by ppalucki — APPROVED** [review]

**Pull request merged**